core/src/epicli/cli/engine/PatchEngine.py

```diff
@@ -1,16 +1,39 @@
 import os
+import copy
+import jsonschema
 
+from cli.version import VERSION
+
+from cli.helpers.Config import Config
+from cli.helpers.Log import Log
 from cli.helpers.Step import Step
+
+from cli.helpers.build_saver import get_inventory_path_for_build
+from cli.helpers.build_saver import copy_files_recursively, copy_file
+from cli.helpers.build_saver import MANIFEST_FILE_NAME
+
+from cli.helpers.yaml_helpers import dump
+from cli.helpers.data_loader import load_yamls_file, load_yaml_obj, types as data_types
+from cli.helpers.doc_list_helpers import select_single, ExpectedSingleResultException
+
+from cli.engine.schema.DefaultMerger import DefaultMerger
 from cli.engine.ansible.AnsibleCommand import AnsibleCommand
 from cli.engine.ansible.AnsibleRunner import AnsibleRunner
-from cli.helpers.Config import Config
-from cli.helpers.build_saver import copy_files_recursively, copy_file, get_inventory_path_for_build
 
 
 class PatchEngine(Step):
```

Comment:
Maybe nitpicking but could we change PatchEngine to BackupRecoveryEngine or even splitting it up in a BackupEngine and RecoveryEngine. PatchEngine is just confusing naming for what it does.

Reply:
I reused what was available already. But I don't see any problem in changing it. Let me work on it! :+1: